test/tests/performance/pbs_preemptperformance.py

```diff
@@ -166,78 +153,67 @@ def test_t2(self):
         a = {'resources_available.ncpus': 4800,
              'resources_available.mem': '1500mb'}
         self.server.create_vnodes('vn', a, 1, self.mom, usenatvnode=True)
-        p = '"express_queue, normal_jobs, queue_softlimits, server_softlimits"'
-        rv = self.scheduler.set_sched_config(
-            {'preempt_prio': p})
-        self.assertTrue(rv)
+        p = '"express_queue, normal_jobs, server_softlimits, queue_softlimits"'
+        self.scheduler.set_sched_config({'preempt_prio': p})
 
         a = {
-            'queue_type': 'e',
+            'queue_type': 'execution',
             'started': 'True',
             'enabled': 'True'
         }
         self.server.manager(MGR_CMD_CREATE, QUEUE, a, 'workq2')
 
         a = {'max_run_res_soft.ncpus': "[u:PBS_GENERIC=2]"}
-        rv = self.server.manager(MGR_CMD_SET, QUEUE, a, 'workq', expect=True)
-        self.assertTrue(rv)
+        self.server.manager(MGR_CMD_SET, QUEUE, a, 'workq', expect=True)
 
-        a = {'max_run_res.mem': "[u:pbsuser=1500mb]"}
-        rv = self.server.manager(MGR_CMD_SET, SERVER, a, expect=True)
-        self.assertTrue(rv)
+        a = {'max_run_res.mem': "[u:" + str(TEST_USER) + "=1500mb]"}
+        self.server.manager(MGR_CMD_SET, SERVER, a, expect=True)
 
         for _ in range(8):
-            j = Job(TEST_USER)
             a = {'Resource_List.select': '1:ncpus=3:mem=90mb',
                  'Resource_List.walltime': 9999}
```

Comment:
a doesn't change in the loop, move up top.

Reply:
Will do